tests/tas-mtls/src/test/java/org/alfresco/rest/mtls/TransformServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package org.alfresco.rest.mtls;
+
+import static org.alfresco.utility.model.FileType.TEXT_PLAIN;
+
+import org.alfresco.rest.MtlsRestTest;
+import org.alfresco.utility.model.FileModel;
+import org.alfresco.utility.model.FolderModel;
+import org.alfresco.utility.model.UserModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@ContextConfiguration ("classpath:alfresco-mtls-context.xml")
+public class TransformServiceTest extends MtlsRestTest
+{
+    private UserModel adminUser;
+
+    @BeforeClass (alwaysRun = true)
+    public void dataPreparation()
+    {
+        adminUser = dataUser.getAdminUser();
+    }
+
+    @Test
+    public void testRenditionWithMTLSEnabledTest()
```

Comment:
I am wondering if there is a way to verify if the mTLS is really enabled. Right now we use the Rest API and even if there is no mTLS is not enabled (or is broken) the test will pass. Maybe `MtlsRestTest` could just check if T-Router is enforcing mTLS?

Reply:
Added